src/service.ts

```diff
@@ -552,6 +552,13 @@ export class Service {
     output.setData(s);
   }
 
+  static openBinaryExplorer(file: File) {
+    const win = window.open("", "Wasm Code Explorer", "toolbar=no,ocation=no,directories=no,status=no,menubar=no,location=no,scrollbars=yes,resizable=yes,width=1024,height=568");
+    win.document.write(`<!DOCTYPE html><html><head><title>WebAssembly Code Explorer</title><meta charset="utf-8"> <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/0.20.12/system.js"></script> <link rel="stylesheet" type="text/css" href="style/code-explorer.css"><link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Mono" rel="stylesheet"></head><body>`);
+    win.document.write(`<div class="toolbar"><div class="title"> WebAssembly Code Explorer</div> <button id="openFile">Open File</button><div class="hidden"><input type="file" id="browseFile"></div></div><div class="container"><div id="dump" class="dump"><div class="addresses"><div></div></div><div class="rows"><div class="row"></div></div><div class="asciis"><div class="ascii"></div></div></div><div id="text" class="text"></div></div> <script>window["rootArr"]=new Uint8Array([${file.data}]);SystemJS.config({map:{\'wasmparser\':\'https://npmcdn.com/wasmparser/\',},packages:{\'wasmparser\':{format:\'cjs\',main:\'./dist/index.js\',defaultExtension:\'js\',map:{\'stream\':\'data:,exports.Transform=function(){}\',},},}});System.import(\'./lib/code-explorer.js\');</script>`);
```

Comment:
This is absolutely barbaric. Any better way to do this?

Reply:
> This is absolutely barbaric. Any other way to do this?

save it as a static file (e.g. code-explorer.html) and open it as a such?